R/create_event.R

```diff
@@ -0,0 +1,30 @@
+#' Create a Google calendar event
+#'
+#' This function allows you to create a calendar event.
+#'
+#'
+#' @param id calendar id, obtained from [calendar_list()]
+#' @param start_time the event start time formatted "yyyy-mm-dd hh:mm:ss". Hours, minutes, seconds, optional.
+#' @param end_time the event start time formatted "yyyy-mm-dd hh:mm:ss". Hours, minutes, seconds, optional.
+#' @param attendees character vector of email addresses of event attendees. Optional.
+#' @param description description of the event. This should be a character vector. Optional.
+#' @param timezone See \code{?timezones}.
+#' @export
+
+create_event <- function(id, attendees, description, start_time, end_time, timezone = Sys.timezone()) {
```

Comment:
I think it will be a better user experience to specify a duration (in minutes) rather than and `end_time`.